examples/python-text2img/handler.py

```diff
@@ -0,0 +1,23 @@
+import requests
+import os
+from bottle import HTTPResponse
+
+def main(event, context):
+
+    text = event["extensions"]["request"].params.get("text")
+
+    if text:
+        url = os.getenv('URL')
+        apiKey = os.getenv('API-KEY')
+
+        print('Trying to query {} with "{}"'.format(url, text))
+
+        # Send request to the external service
+        response = requests.post(url, files=dict(text=text), headers={"api-key": apiKey})
+    
+        return response.json()
+
+    else:
+        return HTTPResponse(body={"'text' parameter is mandatory"}, status=400)
```

Comment:
In Python we can use single or double quote - both values are equal.
In this whole code we can see:
```
"'text' parameter is mandatory"
```
```
"api-key"
```
```
'Trying to query {} with "{}"'
```
```
'URL'
```
IMO will be better if you use quotes in consistent way - always single (or double) quotes on the start and the end of a string.

Reply:
done